person.rb

```diff
@@ -20,6 +20,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Please the `person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association since the `Rental` class' initialize method has handles adding the rental to the `rentals` array in the `Person` class.
See HINT below 👇 
```ruby
def add_rental(book, date)
  Rental.new(date, self, book)
end
```

Reply:
@codepantha The book, and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association as these were the issues to initialize the adding rental have been added and resolved so the working can be done in the proper way.